### ISSUE TITLE
Should we rename array.targetLocales?

### ISSUE THREAD

**Issue by stonea** [chapel-lang/chapel#19031]
Digging through the notes for the Array module review (https://github.com/Cray/chapel-private/issues/2241) I realize we decided that we should open an issue for this (but I forgot to do so at the time so I'm opening it now).

So, here's the question: should we rename `targetLocales`?  We could abbreviate it to `tgtLocales` or name it to `locales`.

We already have a `.locale` property (signifying where an array was declared) so if we name it to `.locales` some users may find this subtle distinction confusing.

Note: we also have a `targetLocales()` function on distributions.  So if we do want to change the nge the name here we may want to consider keeping the names consistent

**Comment by bradcray** [comment]
I don't find `targetLocales` objectionable, but am also used to it.  I definitely prefer it over `tgtLocales` and would probably change to `targetLocs` if I were to abbreviate it at all.  I find `.locales` reasonably attractive, but agree that there's potential for confusion, and am not sure I like it so much that I'd advocate for changing it (and requiring the ensuing code updates from us and from users) just for the sake of changing it.  But if we were defining this for the first time today and that proposal came up, I'd want to discuss it more.

Bottom line: though a little verbose, I don't think this is a name that I'll regret us having in 5 years.

**Comment by lydia-duncan** [comment]
I'm good with `targetLocales` or `targetLocs`.  I found Engin's argument about `.locales` in the meeting quite compelling, so I'd prefer we not use that

**Comment by stonea** [comment]
Sounds like `tgtLocales` is fairly unpopular and I don't think I'd go with `targetLocs` unless there's a precedent of abbreviating locales that way in other places in our standard modules.

I didn't get feedback from anyone advocating for a name change during the module review meeting so unless someone speaks up in the next few days (or some future person wants to revives this issue and pokes me about it) I think I'm going to close it and leave `targetLocales` as-is.

**Comment by stonea** [comment]
Ok, I'm closing this issue. If anyone feels that array.targetLocales should be renamed feel free to revive and bring to discussion.
